tests/FVTTests/homeView/testResizingOfDatasetTree.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+import { expect } from 'chai';
+import { WebDriver, until, By } from "selenium-webdriver"
+
+import { loadPage } from 'explorer-fvt-utilities';
+import { getDriver, TEST_CONFIG } from '../testConfig';
+const { BASE_URL_WITH_PATH } = TEST_CONFIG;
+
+
+// Need to use unnamed function so we can specify the retries
+// eslint-disable-next-line
+describe('JES explorer page load', function () {
+    let driver: WebDriver;
+    this.retries(0);
```

Comment:
As this depends on the jobs loading before tests run I still think we should set the retries the same as other tests which is 3

Reply:
I believe it actually not depends on jobs loading, I added waiting of 'refresh-icon' to be rendered just in case. 

And initially I removed retries as collapse and expand checks are related to each other and switch function just does click action regardless of current sidebar state. 
So retry click could lead to unexpected behaviour, and it could mark test as succeed while it should be actually failed in some cases.

I believe I could rewrite test to ensure some distinct state of app before testing, if needed.

Also I have some issues with passing tests here in the pipeline using exact numbers, while everything works on my instance. I set the window width before test, but exact width numbers still does not match. And I don't know what am I missing. So I had to make tests less accurate. Could it be related to jobs loading and retries? 

Thanks for the review.
Sergei